client/app/admin/admin.component.ts

```diff
@@ -0,0 +1,181 @@
+import { Component, OnInit } from '@angular/core';
+
+import { AdminService } from './admin.service';
+import { MenuService } from '../menu/menu.service';
+
+@Component({
+  selector: 'app-admin',
+  templateUrl: './admin.component.html',
+  styleUrls: ['./admin.component.css']
+})
+export class AdminComponent implements OnInit {
+
+  public menu = [];
+
+  /* Dummy data */
+  public food = [
+      {
+          name: 'Chicken Soup',
+          price: 7,
+          category: 'Food'
+      },
+      {
+          name: 'Curry',
+          price: 11.50,
+          category: 'Food'
+      },
+      {
+          name: 'Hot and Sour Soup',
+          price: 7,
+          category: 'Food'
+      },
+      {
+          name: 'Rice',
+          price: 3.50,
+          category: 'Food'
+      },
+      {
+          name: 'Iced Lemon Tea',
+          price: 3.50,
+          category: 'Drink'
+      },
+      {
+          name: 'CocaCola',
+          price: 2.50,
+          category: 'Drink'
+      },
+      {
+          name: 'Sweet and Sour',
+          price: 6.50,
+          category: 'Food'
+      },
+      {
+          name: 'Ice Cream',
+          price: 3.50,
+          category: 'Food'
+      },
+      {
+          name: 'Plum Wine',
+          price: 4.50,
+          category: 'Drink'
+      },
+      {
+          name: 'Fruit Bowl',
+          price: 3.50,
+          category: 'Food'
+      },
+      {
+          name: 'Stir Fry',
+          price: 7.50,
+          category: 'Food'
+      }
+
+  ];
+
+  public staff = [
+    {
+      firstname: 'Klaus',
+      surname: 'Logan',
+      role: 'Waiter'
+    },
+    {
+      firstname: 'Susanne',
+      surname: 'Smith',
+      role: 'Waiter'
+    },
+    {
+      firstname: 'Joanne',
+      surname: 'Lee',
+      role: 'Manager'
+    },
+    {
+      firstname: 'John',
+      surname: 'McGregor',
+      role: 'Manager'
+    },
+    {
+      firstname: 'Lucy',
+      surname: 'McCabe',
+      role: 'Chef'
+    }
+  ];
+
+  displayOrdersToday_bool;
+  displayOrdersWeek_bool;
+  displayOrdersMonth_bool;
+
+  constructor(
+    protected adminService: AdminService,
+    private menuService: MenuService
+  ) { }
+
+  ngOnInit() {
+    this.menuService.getMenu().subscribe(result => {
+        console.log(result[0].name);
+        this.menu = result;
+    })
+  }
+
+  openMenuNav() {
+    document.getElementById("menuBar").style.width = "20%";
```

Comment:
This would be better achieved with a variable within the component to toggle the open state of the menu, say `menuOpen = false` and then you would toggle classes on the elements within the template via [ngClass](https://angular.io/api/common/NgClass). 

You tend not to want to interact with the DOM nodes directly in the traditional manner as it may have some side effects when angular change detection kicks in and rerenders the node.